src/translib/transformer/xfmr_vrf.go

```diff
@@ -353,7 +353,7 @@ var YangToDb_network_instance_enabled_field_xfmr FieldXfmrYangToDb = func(inPara
         }
 
         if strings.HasPrefix(inParams.key, "Vlan") {
-            log.Infof("YangToDb Vlan key %s, do not add fallback attriubtes.", inParams.key)
+		xfmrLogInfoAll("YangToDb Vlan key %s, do not add fallback attriubtes: %v ", inParams.key)
```

Comment:
extra :%v.
Also, the original log only prints inParams.key, does it need xfmrLogInfoAll?